src/SSHTunnel.php

```diff
@@ -59,9 +58,14 @@ public function createSshTunnel(array $dbConfig): array
         if (empty($sshConfig['sshPort'])) {
             $sshConfig['sshPort'] = self::DEFAULT_SSH_PORT;
         }
-        $sshConfig['privateKey'] = isset($sshConfig['keys']['#private'])
-            ? $sshConfig['keys']['#private']
-            : $sshConfig['keys']['private'];
+
+        if (isset($sshConfig['keys']['#private'])) {
+            $sshConfig['privateKey'] = $sshConfig['keys']['#private'];
+        } else {
+            $sshConfig['privateKey'] = $sshConfig['keys']['private'];
+            $this->logger->warning('Use unencrypted private key!');
```

Comment:
> Configuration is using plaintext private key! Private key should be encrypted. Please contact support if unsure how to do that. 

Co? @ErikZigo @yustme ? Moc?

Reply:
myslim, ze je to jedno, stejne to nebude nikdo cist :)

kde se pouziva unencrypted klic muzem vytahnout z shared configu a podle toho kolik jich je s tim neco udelat

Reply:
OK, tak aspoň teda ať je to pochopitelný. Tj buď "Using unecrypted private key". Takhle je to rozkazovací způsob. "Použij nešifrovaný klíč!" :)

Reply:
je to tam

Reply:
Why do we allow use of an unencrypted key?  Shouldn't it result in a user error instead?

Reply:
backwards compatibility and no one bothered to do a migration for it IMO

Reply:
We can't break functioning configuration out of the blue...

Reply:
hmmm, might be worth doing a check to see if anybody is actually using it

Reply:
https://github.com/keboola/internal/issues/307